### PR TITLE
Added tac-k1 keyboard

### PR DESCRIPTION
**Pull request by itarze**
## Description

Added tac-k1 keyboard.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/tacworks/tac_k1/info.json [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": false,
+        "extrakey": true,
+        "haptic": false,
+        "mousekey": true,
+        "nkro": true,
+        "rgb_matrix": false,
+        "rgblight": false
```
Comment (zvecr) [inline keyboards/tacworks/tac_k1/info.json:17]:
```suggestion
        "extrakey": true,
        "mousekey": true,
        "nkro": true
```

File: keyboards/tacworks/tac_k1/keymaps/via/keymap.c [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        _______, _______, _______,                    _______, _______, _______,                     _______, _______,          _______, _______, _______
+    ),
+
+    [3] = LAYOUT_all(
+        _______, _______, _______, _______, _______,  _______, _______, _______,  _______,  _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______,  _______, _______, _______,  _______,  _______, _______, _______, _______, _______,          _______,
+        _______, _______, _______, _______, _______,  _______, _______, _______,  _______,  _______, _______, _______, _______, _______,          _______,
+        _______, _______, _______, _______, _______,  _______, _______, _______,  _______,  _______, _______, _______,          _______, _______, _______,
+        _______, _______, _______,                    _______, _______, _______,                     _______, _______,          _______, _______, _______
+    )
```
Comment (zvecr) [inline keyboards/tacworks/tac_k1/keymaps/via/keymap.c:37]:
```suggestion
    )
```

File: keyboards/tacworks/tac_k1/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        _______, _______, _______,                    _______, _______, _______,                     _______, _______,          _______, _______, _______
+    ),
+
+    [3] = LAYOUT_all(
+        _______, _______, _______, _______, _______,  _______, _______, _______,  _______,  _______, _______, _______, _______, _______, _______, _______,
+        _______, _______, _______, _______, _______,  _______, _______, _______,  _______,  _______, _______, _______, _______, _______,          _______,
+        _______, _______, _______, _______, _______,  _______, _______, _______,  _______,  _______, _______, _______, _______, _______,          _______,
+        _______, _______, _______, _______, _______,  _______, _______, _______,  _______,  _______, _______, _______,          _______, _______, _______,
+        _______, _______, _______,                    _______, _______, _______,                     _______, _______,          _______, _______, _______
+    )
```
Comment (zvecr) [inline keyboards/tacworks/tac_k1/keymaps/default/keymap.c:37]:
```suggestion
    )
```

File: keyboards/tacworks/tac_k1/info.json [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+    "url": "https://tacworks.store",
+    "usb": {
+        "device_version": "1.0.0",
+        "force_nkro": true,
+        "pid": "0xE431",
+        "suspend_wakeup_delay": 1000,
+        "vid": "0x342D"
+    },
+    "layouts": {
+        "LAYOUT_all": {
```
Comment (zvecr) [inline keyboards/tacworks/tac_k1/info.json:37]: Renders incorrectly

File: keyboards/tacworks/tac_k1/info.json [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+                {"matrix": [4, 6], "x": 6.25, "y": 4, "w": 1.25},
+                {"matrix": [4, 7], "x": 7.5, "y": 4, "w": 2.5},
+                {"matrix": [4, 8], "x": 10, "y": 4, "w": 1.25},
+                {"matrix": [4, 9], "x": 11.25, "y": 4, "w": 1.25},
+                {"matrix": [4, 13], "x": 13, "y": 4},
+                {"matrix": [4, 14], "x": 14, "y": 4},
+                {"matrix": [4, 15], "x": 15, "y": 4}
+            ]
+        },
+        "LAYOUT_ansi": {
```
Comment (zvecr) [inline keyboards/tacworks/tac_k1/info.json:113]: Looks to be `LAYOUT_65_ansi_blocker` community layout, but also renders incorrectly

File: keyboards/tacworks/tac_k1/info.json [hunk 6]
```diff
@@ -0,0 +1,257 @@
...
+                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
+                {"matrix": [4, 8], "x": 10, "y": 4, "w": 1.25},
+                {"matrix": [4, 9], "x": 11.25, "y": 4, "w": 1.25},
+                {"matrix": [4, 13], "x": 13, "y": 4},
+                {"matrix": [4, 14], "x": 14, "y": 4},
+                {"matrix": [4, 15], "x": 15, "y": 4}
+            ]
+        },
+        "LAYOUT_iso": {
```
Comment (zvecr) [inline keyboards/tacworks/tac_k1/info.json:184]: This looks to be `LAYOUT_65_iso_blocker`... and you guessed it, renders incorrectly

**Review by zvecr — CHANGES_REQUESTED** [review]
Also as a FYI, some of the suggestions have been asked on your previous PRs.

File: keyboards/tacworks/tac_k1/info.json (hunk 6 above)
Comment (itarze) [reply keyboards/tacworks/tac_k1/info.json:180]: Sorry, I'm not sure how to deal with this issue.

File: keyboards/tacworks/tac_k1/info.json [hunk 7]
```diff
@@ -0,0 +1,253 @@
...
+                {"matrix": [3, 15], "x": 15, "y": 3},
+                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
+                {"matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
+                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
+                {"matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
+                {"matrix": [4, 8], "x": 10, "y": 4, "w": 1.25},
+                {"matrix": [4, 9], "x": 11.25, "y": 4, "w": 1.25},
+                {"matrix": [4, 13], "x": 13, "y": 4},
+                {"matrix": [4, 14], "x": 14, "y": 4},
+                {"matrix": [4, 15], "x": 15, "y": 4}
```
Comment (dunk2k) [inline keyboards/tacworks/tac_k1/info.json:249]: 1. add Community Layout Support
2. fix layout rendering
```suggestion
    "community_layouts": [
        "65_ansi_blocker",
        "65_ansi_blocker_split_bs",
        "65_iso_blocker",
        "65_iso_blocker_split_bs"
    ],
    "layouts": {
        "LAYOUT_all": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [0, 13], "x": 13, "y": 0},
                {"matrix": [0, 14], "x": 14, "y": 0},
                {"matrix": [0, 15], "x": 15, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [1, 7], "x": 7.5, "y": 1},
                {"matrix": [1, 8], "x": 8.5, "y": 1},
                {"matrix": [1, 9], "x": 9.5, "y": 1},
                {"matrix": [1, 10], "x": 10.5, "y": 1},
                {"matrix": [1, 11], "x": 11.5, "y": 1},
                {"matrix": [1, 12], "x": 12.5, "y": 1},
                {"matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [1, 14], "x": 15, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [2, 7], "x": 7.75, "y": 2},
                {"matrix": [2, 8], "x": 8.75, "y": 2},
                {"matrix": [2, 9], "x": 9.75, "y": 2},
                {"matrix": [2, 10], "x": 10.75, "y": 2},
                {"matrix": [2, 11], "x": 11.75, "y": 2},
                {"matrix": [2, 12], "x": 12.75, "y": 2},
                {"matrix": [2, 13], "x": 13.75, "y": 2, "w": 1.25},
                {"matrix": [2, 14], "x": 15, "y": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 1.25},
                {"matrix": [3, 1], "x": 1.25, "y": 3},
                {"matrix": [3, 2], "x": 2.25, "y": 3},
                {"matrix": [3, 3], "x": 3.25, "y": 3},
                {"matrix": [3, 4], "x": 4.25, "y": 3},
                {"matrix": [3, 5], "x": 5.25, "y": 3},
                {"matrix": [3, 6], "x": 6.25, "y": 3},
                {"matrix": [3, 7], "x": 7.25, "y": 3},
                {"matrix": [3, 8], "x": 8.25, "y": 3},
                {"matrix": [3, 9], "x": 9.25, "y": 3},
                {"matrix": [3, 10], "x": 10.25, "y": 3},
                {"matrix": [3, 11], "x": 11.25, "y": 3},
                {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [3, 14], "x": 14, "y": 3},
                {"matrix": [3, 15], "x": 15, "y": 3},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [4, 5], "x": 3.75, "y": 4, "w": 2.5},
                {"matrix": [4, 6], "x": 6.25, "y": 4, "w": 1.25},
                {"matrix": [4, 7], "x": 7.5, "y": 4, "w": 2.5},
                {"matrix": [4, 8], "x": 10, "y": 4, "w": 1.25},
                {"matrix": [4, 9], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [4, 13], "x": 13, "y": 4},
                {"matrix": [4, 14], "x": 14, "y": 4},
                {"matrix": [4, 15], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_65_ansi_blocker": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [0, 13], "x": 13, "y": 0, "w": 2},
                {"matrix": [0, 15], "x": 15, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [1, 7], "x": 7.5, "y": 1},
                {"matrix": [1, 8], "x": 8.5, "y": 1},
                {"matrix": [1, 9], "x": 9.5, "y": 1},
                {"matrix": [1, 10], "x": 10.5, "y": 1},
                {"matrix": [1, 11], "x": 11.5, "y": 1},
                {"matrix": [1, 12], "x": 12.5, "y": 1},
                {"matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [1, 14], "x": 15, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [2, 7], "x": 7.75, "y": 2},
                {"matrix": [2, 8], "x": 8.75, "y": 2},
                {"matrix": [2, 9], "x": 9.75, "y": 2},
                {"matrix": [2, 10], "x": 10.75, "y": 2},
                {"matrix": [2, 11], "x": 11.75, "y": 2},
                {"matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                {"matrix": [2, 14], "x": 15, "y": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [3, 2], "x": 2.25, "y": 3},
                {"matrix": [3, 3], "x": 3.25, "y": 3},
                {"matrix": [3, 4], "x": 4.25, "y": 3},
                {"matrix": [3, 5], "x": 5.25, "y": 3},
                {"matrix": [3, 6], "x": 6.25, "y": 3},
                {"matrix": [3, 7], "x": 7.25, "y": 3},
                {"matrix": [3, 8], "x": 8.25, "y": 3},
                {"matrix": [3, 9], "x": 9.25, "y": 3},
                {"matrix": [3, 10], "x": 10.25, "y": 3},
                {"matrix": [3, 11], "x": 11.25, "y": 3},
                {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [3, 14], "x": 14, "y": 3},
                {"matrix": [3, 15], "x": 15, "y": 3},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
                {"matrix": [4, 8], "x": 10, "y": 4, "w": 1.25},
                {"matrix": [4, 9], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [4, 13], "x": 13, "y": 4},
                {"matrix": [4, 14], "x": 14, "y": 4},
                {"matrix": [4, 15], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_65_ansi_blocker_split_bs": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [0, 13], "x": 13, "y": 0},
                {"matrix": [0, 14], "x": 14, "y": 0},
                {"matrix": [0, 15], "x": 15, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [1, 7], "x": 7.5, "y": 1},
                {"matrix": [1, 8], "x": 8.5, "y": 1},
                {"matrix": [1, 9], "x": 9.5, "y": 1},
                {"matrix": [1, 10], "x": 10.5, "y": 1},
                {"matrix": [1, 11], "x": 11.5, "y": 1},
                {"matrix": [1, 12], "x": 12.5, "y": 1},
                {"matrix": [1, 13], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [1, 14], "x": 15, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [2, 7], "x": 7.75, "y": 2},
                {"matrix": [2, 8], "x": 8.75, "y": 2},
                {"matrix": [2, 9], "x": 9.75, "y": 2},
                {"matrix": [2, 10], "x": 10.75, "y": 2},
                {"matrix": [2, 11], "x": 11.75, "y": 2},
                {"matrix": [2, 13], "x": 12.75, "y": 2, "w": 2.25},
                {"matrix": [2, 14], "x": 15, "y": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [3, 2], "x": 2.25, "y": 3},
                {"matrix": [3, 3], "x": 3.25, "y": 3},
                {"matrix": [3, 4], "x": 4.25, "y": 3},
                {"matrix": [3, 5], "x": 5.25, "y": 3},
                {"matrix": [3, 6], "x": 6.25, "y": 3},
                {"matrix": [3, 7], "x": 7.25, "y": 3},
                {"matrix": [3, 8], "x": 8.25, "y": 3},
                {"matrix": [3, 9], "x": 9.25, "y": 3},
                {"matrix": [3, 10], "x": 10.25, "y": 3},
                {"matrix": [3, 11], "x": 11.25, "y": 3},
                {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [3, 14], "x": 14, "y": 3},
                {"matrix": [3, 15], "x": 15, "y": 3},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
                {"matrix": [4, 8], "x": 10, "y": 4, "w": 1.25},
                {"matrix": [4, 9], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [4, 13], "x": 13, "y": 4},
                {"matrix": [4, 14], "x": 14, "y": 4},
                {"matrix": [4, 15], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_65_iso_blocker": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [0, 13], "x": 13, "y": 0, "w": 2},
                {"matrix": [0, 15], "x": 15, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [1, 7], "x": 7.5, "y": 1},
                {"matrix": [1, 8], "x": 8.5, "y": 1},
                {"matrix": [1, 9], "x": 9.5, "y": 1},
                {"matrix": [1, 10], "x": 10.5, "y": 1},
                {"matrix": [1, 11], "x": 11.5, "y": 1},
                {"matrix": [1, 12], "x": 12.5, "y": 1},
                {"matrix": [1, 14], "x": 15, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [2, 7], "x": 7.75, "y": 2},
                {"matrix": [2, 8], "x": 8.75, "y": 2},
                {"matrix": [2, 9], "x": 9.75, "y": 2},
                {"matrix": [2, 10], "x": 10.75, "y": 2},
                {"matrix": [2, 11], "x": 11.75, "y": 2},
                {"matrix": [2, 12], "x": 12.75, "y": 2},
                {"matrix": [2, 13], "x": 13.75, "y": 1, "w": 1.25, "h": 2},
                {"matrix": [2, 14], "x": 15, "y": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 1.25},
                {"matrix": [3, 1], "x": 1.25, "y": 3},
                {"matrix": [3, 2], "x": 2.25, "y": 3},
                {"matrix": [3, 3], "x": 3.25, "y": 3},
                {"matrix": [3, 4], "x": 4.25, "y": 3},
                {"matrix": [3, 5], "x": 5.25, "y": 3},
                {"matrix": [3, 6], "x": 6.25, "y": 3},
                {"matrix": [3, 7], "x": 7.25, "y": 3},
                {"matrix": [3, 8], "x": 8.25, "y": 3},
                {"matrix": [3, 9], "x": 9.25, "y": 3},
                {"matrix": [3, 10], "x": 10.25, "y": 3},
                {"matrix": [3, 11], "x": 11.25, "y": 3},
                {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [3, 14], "x": 14, "y": 3},
                {"matrix": [3, 15], "x": 15, "y": 3},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
                {"matrix": [4, 8], "x": 10, "y": 4, "w": 1.25},
                {"matrix": [4, 9], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [4, 13], "x": 13, "y": 4},
                {"matrix": [4, 14], "x": 14, "y": 4},
                {"matrix": [4, 15], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_65_iso_blocker_split_bs": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [0, 1], "x": 1, "y": 0},
                {"matrix": [0, 2], "x": 2, "y": 0},
                {"matrix": [0, 3], "x": 3, "y": 0},
                {"matrix": [0, 4], "x": 4, "y": 0},
                {"matrix": [0, 5], "x": 5, "y": 0},
                {"matrix": [0, 6], "x": 6, "y": 0},
                {"matrix": [0, 7], "x": 7, "y": 0},
                {"matrix": [0, 8], "x": 8, "y": 0},
                {"matrix": [0, 9], "x": 9, "y": 0},
                {"matrix": [0, 10], "x": 10, "y": 0},
                {"matrix": [0, 11], "x": 11, "y": 0},
                {"matrix": [0, 12], "x": 12, "y": 0},
                {"matrix": [0, 13], "x": 13, "y": 0},
                {"matrix": [0, 14], "x": 14, "y": 0},
                {"matrix": [0, 15], "x": 15, "y": 0},
                {"matrix": [1, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [1, 1], "x": 1.5, "y": 1},
                {"matrix": [1, 2], "x": 2.5, "y": 1},
                {"matrix": [1, 3], "x": 3.5, "y": 1},
                {"matrix": [1, 4], "x": 4.5, "y": 1},
                {"matrix": [1, 5], "x": 5.5, "y": 1},
                {"matrix": [1, 6], "x": 6.5, "y": 1},
                {"matrix": [1, 7], "x": 7.5, "y": 1},
                {"matrix": [1, 8], "x": 8.5, "y": 1},
                {"matrix": [1, 9], "x": 9.5, "y": 1},
                {"matrix": [1, 10], "x": 10.5, "y": 1},
                {"matrix": [1, 11], "x": 11.5, "y": 1},
                {"matrix": [1, 12], "x": 12.5, "y": 1},
                {"matrix": [1, 14], "x": 15, "y": 1},
                {"matrix": [2, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [2, 1], "x": 1.75, "y": 2},
                {"matrix": [2, 2], "x": 2.75, "y": 2},
                {"matrix": [2, 3], "x": 3.75, "y": 2},
                {"matrix": [2, 4], "x": 4.75, "y": 2},
                {"matrix": [2, 5], "x": 5.75, "y": 2},
                {"matrix": [2, 6], "x": 6.75, "y": 2},
                {"matrix": [2, 7], "x": 7.75, "y": 2},
                {"matrix": [2, 8], "x": 8.75, "y": 2},
                {"matrix": [2, 9], "x": 9.75, "y": 2},
                {"matrix": [2, 10], "x": 10.75, "y": 2},
                {"matrix": [2, 11], "x": 11.75, "y": 2},
                {"matrix": [2, 12], "x": 12.75, "y": 2},
                {"matrix": [2, 13], "x": 13.75, "y": 1, "w": 1.25, "h": 2},
                {"matrix": [2, 14], "x": 15, "y": 2},
                {"matrix": [3, 0], "x": 0, "y": 3, "w": 1.25},
                {"matrix": [3, 1], "x": 1.25, "y": 3},
                {"matrix": [3, 2], "x": 2.25, "y": 3},
                {"matrix": [3, 3], "x": 3.25, "y": 3},
                {"matrix": [3, 4], "x": 4.25, "y": 3},
                {"matrix": [3, 5], "x": 5.25, "y": 3},
                {"matrix": [3, 6], "x": 6.25, "y": 3},
                {"matrix": [3, 7], "x": 7.25, "y": 3},
                {"matrix": [3, 8], "x": 8.25, "y": 3},
                {"matrix": [3, 9], "x": 9.25, "y": 3},
                {"matrix": [3, 10], "x": 10.25, "y": 3},
                {"matrix": [3, 11], "x": 11.25, "y": 3},
                {"matrix": [3, 12], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [3, 14], "x": 14, "y": 3},
                {"matrix": [3, 15], "x": 15, "y": 3},
                {"matrix": [4, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [4, 1], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [4, 2], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [4, 6], "x": 3.75, "y": 4, "w": 6.25},
                {"matrix": [4, 8], "x": 10, "y": 4, "w": 1.25},
                {"matrix": [4, 9], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [4, 13], "x": 13, "y": 4},
                {"matrix": [4, 14], "x": 14, "y": 4},
                {"matrix": [4, 15], "x": 15, "y": 4}
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/tacworks/tac_k1/readme.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
+
+# TAC-K1
+
+![tack-k1](https://i.imgur.com/3YGvrSZ.jpg)
```
Comment (dunk2k) [inline keyboards/tacworks/tac_k1/readme.md:4]:
```suggestion
![tack-k1](https://i.imgur.com/3YGvrSZh.jpg)
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

File: keyboards/tacworks/tac_k1/readme.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
+
+# TAC-K1
+
+![tack-k1](https://i.imgur.com/3YGvrSZh.jpg)
```
Comment (dunk2k) [inline keyboards/tacworks/tac_k1/readme.md:4]:
```suggestion
![tac-k1](https://i.imgur.com/3YGvrSZh.jpg)
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/tacworks/tac_k1/rules.mk [hunk 10]
```diff
@@ -0,0 +1 @@
+WEAR_LEVELING_DRIVER = spi_flash
```
Comment (zvecr) [inline keyboards/tacworks/tac_k1/rules.mk:1]:
```suggestion
# This file intentionally left blank
```

File: keyboards/tacworks/tac_k1/config.h [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+
+/* SPI Config for spi flash*/
+#define SPI_DRIVER SPIDQ
+#define SPI_SCK_PIN B3
+#define SPI_MOSI_PIN B5
+#define SPI_MISO_PIN B4
+#define SPI_MOSI_PAL_MODE 5
+
+#define EXTERNAL_FLASH_SPI_SLAVE_SELECT_PIN C12
+#define WEAR_LEVELING_BACKING_SIZE (4 * 1024)
```
Comment (zvecr) [inline keyboards/tacworks/tac_k1/config.h:14]:
```suggestion
```

File: keyboards/tacworks/tac_k1/info.json [hunk 12]
```diff
@@ -0,0 +1,404 @@
+{
+    "manufacturer": "TACWORKS",
+    "keyboard_name": "TAC-K1",
+    "maintainer": "JoyLee",
+    "bootloader": "wb32-dfu",
+    "diode_direction": "ROW2COL",
+    "features": {
```
Comment (zvecr) [inline keyboards/tacworks/tac_k1/info.json:7]:
```suggestion
    "eeprom": {
        "driver": "wear_leveling",
        "wear_leveling": {
            "driver": "spi_flash",
            "backing_size": 4096
        }
    },
    "features": {
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Comment by itarze** [comment]
Can merge？

**Review by fauxpark — APPROVED** [review]

**Pull request merged**